doc/nrf/releases/release-notes-1.8.0.rst

```diff
@@ -0,0 +1,832 @@
+    A-GPS support is enabled by default.
+  * Added PPP-related updates:
+
+    * Added IPv6 support.
+    * Added LTE link MTU to be informed to PC.
+    * Improved autostart of PPP.
+    * Made changes for better performance.
+
+* :ref:`gnss_sample` sample:
+
+  * Added support for periodic fixes.
+  * Added support for power saving.
+  * Added support for low accuracy fixes.
+  * Renamed.
+    The previous name was "nRF9160: GPS with SUPL client library".
+  * Added support for nRF Cloud A-GPS and P-GPS.
+  * LTE now remains connected to the network all the time when assistance is enabled.
+    With A-GPS, the sample can be configured to connect to network only when needed.
+
+* nRF9160: A-GPS sample:
+
+  * The sample has been removed.
+    nRF Cloud A-GPS and P-GPS are demonstrated in the :ref:`gnss_sample` sample.
+
+OpenThread samples
+------------------
+
+* Added:
+
+  * Support for ``nrf5340dk_nrf5340_cpuapp_ns`` build target for :ref:`zephyr:nrf5340dk_nrf5340`.
+    This allows to build the OpenThread samples with Trusted Firmware-M and the PSA crypto API support.
+    This platform is experimental, so :ref:`nrfxlib:ot_libs` are not generated for it.
+
+Zigbee samples
+--------------
+
+* Added:
+
+   * :ref:`Zigbee shell <zigbee_shell_sample>` sample.
+
+* Updated:
+
+   * Fixed issue with cluster declaration in :ref:`Zigbee shell <zigbee_shell_sample>` sample and :ref:`Zigbee template <zigbee_template_sample>` sample.
+
+Other samples
+-------------
+
+* Added the :ref:`hw_unique_key_usage` sample.
+* Added the :ref:`caf_preview_sample` sample.
+* :ref:`bootloader` sample:
+
+  * Improved how hardware unique keys are handled.
+
+    * Introduced :kconfig:`CONFIG_HW_UNIQUE_KEY_LOAD` with fewer dependencies than :kconfig:`CONFIG_HW_UNIQUE_KEY` solely for loading the key.
+    * The bootloader now allows a single boot with no key present, to allow the app to write a key.
+      After the first boot, the key must be present or the bootloader won't boot the app.
+
+  * Bug fixes:
+
+    * Fixed the NCSDK-10209 issue with alignment errors caused by changes in the size of partitions.
+      The |NSIB| and MCUboot have been made more robust against such errors.
+
+* :ref:`radio_test` sample:
+
+  * Clarified units for numerical parameters in shell commands.
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+* Added the API documentation and conceptual documentation for the following drivers:
+
+  * :ref:`sensor_sim`
+  * :ref:`paw3212`
+  * :ref:`pmw3360`
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Bluetooth libraries
+-------------------
+
+* Added the :ref:`lib_hrs_client_readme` library.
+* :ref:`ble_rpc` library:
+
+  * Added support for the GATT API serialization.
+  * Changed the configuration option that enables the library from the :kconfig:`CONFIG_BT_RPC` to the :kconfig:`CONFIG_BT_RPC_STACK`.
+
```

Comment:
Please add:
":ref:`bms_readme` - changed security permissions of the service's characteristics to require authentication." per https://github.com/nrfconnect/sdk-nrf/pull/5854